MaxText/train.py

```diff
@@ -211,13 +212,30 @@ def train_loop(config, state=None):
   Returns:
 
   """
+
+  monitoring_enabled = config.enable_cloud_monitoring
+
+  if monitoring_enabled:
+    monitoring_api.create_custom_metric('checkpointing_init_start', "Checkpointing Initialization Start")
```

Comment:
These metric creations should be hidden inside of a function inside of maxutils, something like "register_train_metrics"

Reply:
Done